.gitignore

```
@@ -0,0 +1,2 @@
+.DS_Store
+.idea/
```

```suggestion
.idea/

```
Please add a new line

---

Addressed in [6a35df4](https://github.com/luisggois/show-git-functionality/pull/1/commits/6a35df4c73bc13df8805fe97beb10ca60afabaff)